docs/_scripts/prep_docs.py

```diff
@@ -5,16 +5,25 @@
 """
 import sys
 from pathlib import Path
+from importlib.metadata import version
+
+from packaging.version import parse
 
 DOCS = Path(__file__).parent.parent.absolute()
 NPE = DOCS.parent.absolute() / 'npe2'
 
 def prep_npe2():
     #   some plugin docs live in npe2 for testing purposes
+    if NPE.exists():
+        return
     from subprocess import check_call
 
+    npe2_version = version("npe2")
+
     check_call(f"rm -rf {NPE}".split())
     check_call(f"git clone https://github.com/napari/npe2 {NPE}".split())
+    if not parse(npe2_version).is_devrelease:
+        check_call(f"git checkout tags/v{npe2_version}".split(), cwd="npe2")
```

Comment:
```suggestion
        check_call(f"git checkout tags/v{npe2_version}".split(), cwd=NPE)
```

This may be the same directory, but it was not clear to me.